pkg/api-server/resource_ws_test.go

```diff
@@ -214,6 +214,10 @@ var _ = Describe("Resource WS", func() {
 			// then
 			Expect(response.StatusCode).To(Equal(400))
 		})
+
+		It("should return 400 on validation error", func() {
```

Comment:
What is this ?

Reply:
placeholder for test, wanted to push this asap to get feedback on the solution itself. Now it's implemented.